sections/next-story.liquid

```diff
@@ -0,0 +1,22 @@
+<div class="next-story">
+    <p class="next-story__title">LOREM IPSUM DOLOR SUM</p>
+    <p class="next-story__body">
+            Vestibulum rutrum quam vitae fringilla tincidunt. Suspendisse nec tortor urna.
+            Ut laoreet sodales nisi, quis iaculis nulla iaculis vitae. Donec sagittis faucibus 
+            lacus eget blandit. Mauris vitae ultricies metus, at condimentum nulla. Donec quis ornare lacus. 
+            Etiam gravida mollis tortor quis porttitor. </p>
+</div>
+<img class="img-in-next-story" src="{{section.settings.img_next_story | img_url: '800x'}}">
+
+{% schema %}
+{
+  "name": "img-next-story",
+  "settings": [
+    {
+      "type": "image_picker",
+      "id": "img_next_story",
+      "label": "img next story"
+    }
+  ]
+}
+{% endschema %}
```

Comment:
the same about the final empty line

Reply:
You're commenting on the same section here https://github.com/JaimeGonz/lil-peep-v20-01/pull/10#discussion_r398845430